src/components/NcInputField/NcInputField.vue

```diff
@@ -441,7 +441,7 @@ export default {
 		margin-inline: var(--input-padding-start) var(--input-padding-end);
 		max-width: fit-content;
 		font-size: var(--input-label-font-size);
-		inset-block-start: calc((var(--default-clickable-area) - var(--default-line-height)) / 2); // center the label vertically
+		inset-block-start: calc((var(--default-clickable-area) - var(--default-line-height) * var(--input-label-font-size)) / 2); // center the label vertically
```

Comment:
It is not compatible with Nextcloud 28..29 with unit line height

Reply:
Couldn't find a way to transform it to another unit, but as `lh` is working, shouldn't be a problem?